pipeline/lib/custom_dockerfile.ml

```diff
@@ -95,12 +95,34 @@ let install_opam_dependencies ~files =
       @@ run "opam exec -- opam install -y --depext-only %s" targets
       @@ run "opam exec -- opam install -y --deps-only --with-test %s" targets
 
+let contains_at s1 s2 =
+  let re = Str.regexp_string s2 in
+  try
+    let idx = Str.search_forward re s1 0 in
+    idx
+  with Not_found -> -1
```

Comment:
Nitpick: I'd rather use `int option` here than using the somewhat obscure (and not type-safe) `-1` to signal something went wrong.

Reply:
Thanks for catching this. I've pushed a change to fix this.